.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Report Bug
+about: Report a bug you have found
+labels:
+  - type/bug
+  - needs-triage
+---
+
+<!-- Please include a self-contained copy-pastable example that generates the issue if possible.
+
+Please be concise with code posted. See guidelines below on how to provide a good bug report:
+
+- Craft Minimal Bug Reports http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports
+- Minimal Complete Verifiable Examples https://stackoverflow.com/help/mcve
+
+Bug reports that follow these guidelines are easier to diagnose, and so are often handled much more quickly.
+-->
```

Comment:
Do we want a prompt for version info?

Reply:
Added!

Reply:
I'm not familiar with issue template syntax - how will this commented out section render to the contributor?

Reply:
They'll see it in the raw version of the markdown where they are typing, but it doesn't show up once rendered (in preview, or after they hit submit)